### PR TITLE
Add mailmap

### PR DESCRIPTION
**Pull request by PeterJCLaw**

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Antoine Petty <a.petty@ncl.ac.uk> <36314396+antoinepetty@users.noreply.github.com>
+Andrew Barrett-Sprot <abarrettsprot@gmail.com> <Adimote@users.noreply.github.com>
+Jake Howard <git@theorangeone.net> <RealOrangeOne@users.noreply.github.com>
+Jimmy Thompson <jimmy@jimmythompson.co.uk>
+Jimmy Thompson <jimmy@jimmythompson.co.uk> <jimmy.thompson@mendeley.com>
+Jimmy Thompson <jimmy@jimmythompson.co.uk> <jimmy.thompson@thoughtworks.com>
+Rob Spanton <rob@robspanton.com> <rob@adventurousmachines.com>
```
Comment (trickeydan) [inline .mailmap]: Didn't Rob change his name?

Comment (PeterJCLaw) [reply .mailmap]: I think he did, though my aim here was to collapse the names in the history rather than to do that renaming.

File: .mailmap [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Antoine Petty <a.petty@ncl.ac.uk> <36314396+antoinepetty@users.noreply.github.com>
```
Comment (antoinepetty) [inline .mailmap:1]:
```suggestion
Antoine Petty <github@antoinepetty.co.uk>
Antoine Petty <github@antoinepetty.co.uk> <36314396+antoinepetty@users.noreply.github.com>
```

**Review by antoinepetty — CHANGES_REQUESTED** [review]

File: .mailmap (hunk 2 above)
Comment (PeterJCLaw) [reply .mailmap]: 791e20b

**Review by antoinepetty — APPROVED** [review]

**Pull request merged**